### PR TITLE
Prepare for merge testing

### PR DESCRIPTION
**Pull request by joshmoore**
- .travis.yml: move logic from to Makefile
 - .travis.yml: docker is now part of "slow" (after gh-543 breakage)
 - make: all is now an alias for "fast"
 - make: slow should be run for master and merge

### CODE REVIEW

**Comment by joshmoore** [comment]
Required step for testing this will be to push a merge branch and see it pass.

**Comment by joshmoore** [comment]
Script used to push merge:

```
TRAVIS_BRANCH=${TRAVIS_BRANCH:-master}
SCC_ARGS=${SCC_ARGS:-"-D all"}
SCC_MISC=${SCC_MISC:-"--no-ask --reset"} # add --comment
SCC_PUSH=${SCC_PUSH:-"--push=merge"}
SCC_STAT=${SCC_STAT:-"-Snone"}
SCC_VENV=${SCC_VENV:-"/tmp/scc"}

set -eux

test -e ${SCC_VENV} || virtualenv ${SCC_VENV}
VIRTUAL_ENV_DISABLE_PROMPT=true source ${SCC_VENV}/bin/activate
pip install scc
scc merge     \
  ${SCC_ARGS} \
  ${SCC_MISC} \
  ${SCC_PUSH} \
  ${SCC_STAT} \
  ${TRAVIS_BRANCH} "$@"
```

Currently failing in https://travis-ci.org/spacetx/starfish/builds/427576260 due to:

```
[ -e notebooks/py/MERFISH_Pipeline_-_U2O2_Cell_Culture_-_1_FOV.py.skip ] || python3.6 notebooks/py/MERFISH_Pipeline_-_U2O2_Cell_Culture_-_1_FOV.py
  File "notebooks/py/MERFISH_Pipeline_-_U2O2_Cell_Culture_-_1_FOV.py", line 96
    dpsf.run(primary_image, verbose=True, n_processes=)
                                                      ^
SyntaxError: invalid syntax
make: *** [run__notebooks/py/MERFISH_Pipeline_-_U2O2_Cell_Culture_-_1_FOV.py] Error 1
```

**Comment by joshmoore** [comment]
This will likely conflict with gh-550 when the time comes.

File: Makefile [hunk 1]
```diff
@@ -23,4 +25,14 @@ docs-%:
 
 include notebooks/subdir.mk
 
-.PHONY: all lint lint-non-init lint-init test mypy
+slow: fast run_notebooks docker
+	virtualenv -p python .venv
```
Comment (ttung) [inline Makefile]: virtualenv is not guaranteed to be installed.  also, `.venv` is where i stored my virtualenv.  can you make this:

```
python -m venv .notebooks-exec-env
.notebooks-exec-env/bin/pip install -r REQUIREMENTS-NOTEBOOK.txt
.notebooks-exec-env/bin/pip install starfish
make PYTHON=.notebooks-exec-env/bin/python run_notebooks
```

**Comment by joshmoore** [comment]
Hmm.... think I'm overlooking a way to re-request a review, so here's a @ttung ping.

**Pull request merged**